spark/src/main/java/org/apache/zeppelin/spark/SparkInterpreter.java

```diff
@@ -604,7 +604,18 @@ public InterpreterResult interpretInput(String[] lines) {
     out.reset();
     Code r = null;
     String incomplete = "";
-    for (String s : linesToRun) {
+
+    for (int l = 0; l < linesToRun.length; l++) {
+      String s = linesToRun[l];      
+      // check if next line starts with "." (but not ".." or "./") it is treated as an invocation
```

Comment:
My understanding is Scala allows line wrapping if it is enclosed by parenthesis or ended with an operator?
`Scala requires each “wrap line” to either have an unclosed parenthetical or to end with an infix method in which the right parameter is not given`
http://docs.scala-lang.org/style/indentation.html

Does it need to be accounted for here?